pkg/services/searchV2/index.go

```diff
@@ -255,9 +271,9 @@ func (i *searchIndex) run(ctx context.Context, orgIDs []int64, reIndexSignalCh c
 				defer func() { <-asyncReIndexSemaphore }()
 
 				started := time.Now()
-				i.logger.Info("Start re-indexing")
-				i.reIndexFromScratch(ctx)
-				i.logger.Info("Full re-indexing finished", "fullReIndexElapsed", time.Since(started))
+				i.logger.Info("Start re-indexing", i.withCtxData(fullReindexCtx)...)
+				i.reIndexFromScratch(fullReindexCtx)
+				i.logger.Info("Full re-indexing finished", i.withCtxData(fullReindexCtx, "fullReIndexElapsed", time.Since(started))...)
```

Comment:
Based on discussion with @bergquist could be a good convention to use `duration` as key to be consistent. However, today we normally filter logs before getting a rate of `duration` and currently there are no usage of logs without filter getting a rate of `duration` of "everything", but might be in the future. 

In your case, using `duration` and an additional key `reIndexType` to describe `fullReIndex` instead of `fullReIndexElapsed` might be considered. WDYT?

Reply:
I have no strong opinion on this, I prefer non-unique unique labels cause
1. I still have to filter the logs based on something, so its
   - 1. filter `myUniqueKey`, 2. unwrap `myUniqueKey`, or
   - 1. filter `myUniqueKey`, 2. unwrap `duration`
2. I'd need to split a single log into multiple logs even if it keeping it all together makes sense (example - https://github.com/grafana/grafana/blob/74158ed66ba88a1bbb62a772a0343973c63c4ab2/pkg/services/searchV2/index.go#L460-L465) 
3. Finally, iiuc, `unwrap 'duration'` has more potential for including unwanted data if the filtering wasn't exact


But again, no strong opinion, I am a newb at loki :) If you want to standardize and use `duration` ill change it

Reply:
Ohh I see you have multiple durations on a single log there. Well, please skip changing to `duration` then for now.